internal/out/print.go

```diff
@@ -34,6 +35,7 @@ func Printf(ctx context.Context, format string, args ...interface{}) {
 	if ctxutil.IsHidden(ctx) {
 		return
 	}
+	debug.LogN(1, format, args...)
```

Comment:
I'm not too fond of this since it would add plaintext passwords to the debug log.

Reply:
This shouldn't change what we log. It's supposed to only fix the originator shown in the log.
Why do you believe this will make us start logging plaintext passwords?

(Note: Yes, the debug can contain these, and yes we might want to change that. But this shouldn't be affected by this PR).

Reply:
Because of https://github.com/gopasspw/gopass/blob/456c18a1c2f751746ce5900107fa1f62dbd4332c/internal/action/show.go#L174-L175
and `Print` is just using `Printf` behind the hood.

Reply:
Ack. This needs to be changed. But it's not the only place where we log secrets. The parser has a bunch of debug statements as well.